### PR TITLE
Converted Conveyor and TwoWayLever Visualizers to ECS format

### PR DESCRIPTION
**Pull request by ModeratelyAware**
Part of #8995
Converted ConveyorVisualizer to ECS component andsystem.
Converted TwoWayLeverVisualizer to ECS component and system.

EDIT: Removed unnecessary information. Received answers in #8995.

### CODE REVIEW

**Comment by Zumorica** [comment]
Changelogs are meant to be player-facing, they should only include gameplay changes and the like.

**Comment by ModeratelyAware** [comment]
> Changelogs are meant to be player-facing, they should only include gameplay changes and the like.

Ah, okay. Pretty new and it was a template.

**Comment by Zumorica** [comment]
> Ah, okay. Pretty new and it was a template.

No worries! The template also has some useful comments on how best to write changelogs, give it a read next time if you want

**Comment by ModeratelyAware** [comment]
> > Ah, okay. Pretty new and it was a template.
> 
> No worries! The template also has some useful comments on how best to write changelogs, give it a read next time if you want

Thank you! I definitely will!

**Comment by ModeratelyAware** [comment]
Realized the TwoWayLever conversion was actually not working. Will submit it separately.

**Comment by ModeratelyAware** [comment]
Well.. Re-opening this. Realized I shouldn't have closed it and should have just worked on it directly till it passed. :sweat_smile:

File: Resources/Prototypes/Entities/Structures/Wallmounts/switch.yml [hunk 1]
```diff
@@ -92,16 +92,21 @@
     - type: Sprite
       netsync: false
       sprite: Structures/conveyor.rsi
-      state: switch-off
+      layers:
+        - state: switch-off
+          map: ["enabled", "enum.TwoWayLeverState.Middle"]
+
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Entities/Structures/Wallmounts/switch.yml:98]:
```suggestion
```

**Review by EmoGarbage404 — APPROVED** [review]
very good cleanup, it makes me happy inside.

**Comment by ModeratelyAware** [comment]
> very good cleanup, it makes me happy inside.

Phew, thanks! 👍

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**